pkg/database/sql.go

```diff
@@ -0,0 +1,119 @@
+package database
+
+import (
+	"context"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"os"
+)
+
+type SQLClient struct {
+	Client *gorm.DB
+	Logger *log.Entry
+	ctx    context.Context
+	stop   chan os.Signal
+}
+
+type Fileurl struct {
+	Id 			int		`gorm:"primaryKey;column:id"`
+	Url			string	`gorm:"column:url"`
+	Rawdata		string	`gorm:"column:rawdata"`
+}
+
+type FileurlChilds struct {
+	FileurlID 			int		`gorm:"column:fileurl_id"`
+	ChildfileurlId		int		`gorm:"column:childfileurl_id"`
+}
+
+// SetDeps sets dependencies for a parent
+func (c *SQLClient) SetDeps(parent string, deps []string) {
```

Comment:
How feasible is it to write tests for this before end of sprint? If not, how do you feel about a card next sprint just for this work?

Reply:
Tests?

Reply:
![tests](https://media1.tenor.com/images/1bb97c0f5564adf2f40f966bd8851f2e/tenor.gif?itemid=15083309)